packages/fluentui/docs/package.json

```diff
@@ -58,17 +58,22 @@
     "@types/react-custom-scrollbars": "^4.0.5",
     "@types/react-router-dom": "^5.1.5",
     "@types/react-virtualized": "^9.21.8",
-    "@types/react-window": "^1.8.2"
+    "@types/react-window": "^1.8.2",
+    "storywright": "0.0.26-beta.1"
   },
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "build": "gulp build:docs",
+    "build:storybook": "build-storybook -o dist/storybook",
     "lint": "eslint --ext .js,.ts,.tsx .",
     "lint:fix": "yarn lint --fix",
     "start": "gulp docs",
+    "screener:build": "yarn build:storybook",
```

Comment:
Using the name `screener` here to simplify and align with current [template ](https://github.com/microsoft/fluentui/blob/50acef3202bac5df6679a3a9859aebaf2adeb264/.devops/templates/runpublishvrscreenshot.yml#L51)we have while running the VR tool. Will be renamed (along with others) once we completely remove screener.